docs/book/src/plugins/monitoring-v1-alpha.md

```diff
@@ -0,0 +1,58 @@
+# Monitoring Plugin (`monitoring/v1-alpha`)
+
+The Monitoring plugin is an optional plugin that can be used to scaffold Prometheus based monitoring, will provide best practices and tooling for monitoring requirements and help with standardizing the way monitoring is implemented in operators.
+
+<aside class="note">
+<h1>Examples</h1>
+
+You can check its default scaffold by looking at the `project-v3-with-monitoring` projects under the [testdata][testdata] directory on the root directory of the Kubebuilder project.
+
+</aside>
+
+## When to use it ?
+
+- If you are looking to observe the metrics exported by [controller metrics][controller-metrics] and collected by Prometheus via [Monitoring][monitoring].
+
+## How to use it ?
+
+### Prerequisites:
+
+- Your project must be using [controller-runtime][controller-runtime] to expose the metrics via the [controller default metrics][controller-metrics] and they need to be collected by Prometheus.
```

Comment:
@avlitman Please remove. This is not a prerequisite in this use case. Its only relevant to memchached operator.